src/haversine.jl

```diff
@@ -9,6 +9,7 @@ The computed distance has the same units as that of the radius.
 struct Haversine{T<:Real} <: Metric
```

Comment:
The docstring above should mention the default, say what it corresponds to exactly, and give links to references (e.g. Wikipedia).

Reply:
done